api/v1alpha1/groupversion_info.go

```diff
@@ -26,7 +26,7 @@ import (
 
 var (
 	// GroupVersion is group version used to register these objects
-	GroupVersion = schema.GroupVersion{Group: "operator.openshift.io", Version: "v1alpha1"}
+	GroupVersion = schema.GroupVersion{Group: "externaldns.olm.openshift.io", Version: "v1alpha1"}
```

Comment:
The API group changes in this PR will be taken care of via https://github.com/openshift/external-dns-operator/pull/12, right? (I dont see any commits from that PR explicitly cherry-picked into this PR so just wanted to double check)

Reply:
Yeah, you are right. I put a note in the description that the API changes are here just to match the future names. Taken that I renamed the files in `manifests/` to `externaldns...` I thought that it would be good to try the installation of the operator with the relevant API group.
This PR will wait for the merge of #12 anyway so that I will be able to rebase.

Reply:
Ack, thanks for confirming.